src/store/modules/app/banner.js

```diff
@@ -0,0 +1,25 @@
+export default {
+  namespaced: true,
+  state: {
+    component: null,
+    data: {},
```

Comment:
The mobile banner doesn't use this state, right? I think it'd be better to add this later if we need to rather than add it now.

Reply:
@jkeat, 

Yup, that's right `data` isn't being used at all - I'll remove it!